scripts/build.py

```diff
@@ -605,7 +605,7 @@ def install_sdk(self):
                 host = 'darwin' # our packages are still called darwin
 
             # Android NDK
-            download_sdk(self, '%s/%s-%s-x86_64.tar.gz' % (self.package_path, PACKAGES_ANDROID_NDK, host), join(sdkfolder, PACKAGES_ANDROID_NDK))
```

Comment:
the original file on google's side doesn't use`-x86_64` postfix, I removed it on our side as well